tests/jobs/test_local_task_job.py

```diff
@@ -87,6 +87,11 @@ def set_instance_attrs(self, dagbag):
         with patch('airflow.jobs.base_job.sleep') as self.mock_base_job_sleep:
             yield
 
+    def end_process(self, process):
+        process.join(timeout=10)
+        process.terminate()
+        time.sleep(0.1)
```

Comment:
`time.sleep(0.1)` was used because of the example here: https://docs.python.org/3/library/multiprocessing.html#multiprocessing.Process.close